views/page-tou.pug

```diff
@@ -30,10 +30,22 @@ block content
 			li Encouraging/glorifying harm or self-harm of others, including malicious, directed use of "die", "kys", etc. If you or someone you know is at risk of self-harm, #[a(href="https://en.wikipedia.org/wiki/List_of_suicide_crisis_lines") please seek help from a professional or the authorities.]
 			h4
 				strong Explicitly Forbidden Words:
-			li N word -- likely ban on sight
-			li F slur -- likely timeout on sight
-			li &quot;R*tard&quot;
-			li Words that are equivalent to those listed above (e.g., &quot;libtard&quot; for &quot;r*tard&quot;).
+			li You can see the
+				em  uncensored
+				span  word by clicking on the "Click to see"
+			br
+			li N word -- likely ban on sight --
+				p(class="revealer" onclick="toggle('nword')") Click to see:
```

Comment:
I think a single `show all` would be better

Reply:
> Looks good generally. I'd prefer if you didn't LF unchanged files, for now, I'll make a change soon to convert everything to LF to not have confusion with what exactly PRs did.

That was automatically done by my editor and I didn't realize until I PRed it, but I can correct that.